README.md

```diff
@@ -30,7 +30,7 @@ Looking for something specific? 🔍
 
 [{{back to navigation}}](#table-of-contents)
 
-Initially you will need to install [Ruby <2.7](https://www.ruby-lang.org/en/documentation/installation/) and [bundler](https://bundler.io/) with `gem install bundler`.
+Initially you will need to install [Ruby <2.7](https://www.ruby-lang.org/en/supportation/installation/) and [bundler](https://bundler.io/) with `gem install bundler`.
```

Comment:
I accept that "supportation" is a word, but it's a terrible word, I don't think we should use it - it sounds made up. Can we just say support or SUPPORT ?